src/App.js

```diff
@@ -15,10 +15,14 @@ import Profile from "./components/Profile/Profile.component";
 import WishesContainer from "./components/Wishes/WishesContainer";
 import {Routers} from "./AppContainer";
 import {BottomAnchor, GoBottom, GoTop, TopAnchor} from "./common/QuickScrolls/QuickScrolls.component";
+import {GiHamburgerMenu as Hamburger} from "react-icons/gi";
+import {LeftBtn} from "./components/Sidebar/Sidebar.style";
```

Comment:
Don't import button style from other styles except this file name

Reply:
fixed